substrate/frame/support/procedural/src/lib.rs

```diff
@@ -412,79 +411,6 @@ pub fn require_transactional(attr: TokenStream, input: TokenStream) -> TokenStre
 		.unwrap_or_else(|e| e.to_compile_error().into())
 }
 
-/// Derive [`Clone`] but do not bound any generic. Docs are at `frame_support::CloneNoBound`.
-#[proc_macro_derive(CloneNoBound)]
-pub fn derive_clone_no_bound(input: TokenStream) -> TokenStream {
```

Comment:
If we are moving these, we should move them into a complete separate crate that is living outside of the repo etc. IMO this makes this reusable by more people.